apps/docs/content/components/Grid.mdx

```diff
@@ -0,0 +1,274 @@
+
+```js
+import {Grid} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The `Grid` component uses a 12-column layout system. Use `span` to control the number of columns that should be displayed.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column span={1}></Grid.Column>
+  <Grid.Column span={11}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+</Grid>
+```
+
+### Nested grids
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column>
+    <Grid enableOverlay>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+    </Grid>
+  </Grid.Column>
+</Grid>
+```
+
+### Responsive behavior
+
+Use `start` with an `Object` of breakpoint-specific keys and `number` values to enable responsive behavior.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+  <Grid.Column
+    span={{xsmall: 6, small: 6, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+  ></Grid.Column>
+</Grid>
+```
+
+### Column offset
+
+Use `start` to provide a positioning offset.
```

Comment:
In the provided example, it's unclear why specifying only a `start` prop, makes the second column a span=6. AFAICT, the default span is 12?

Reply:
Yes, that's correct it's 12. It's documented in the prop table, but I could update this example to add span={12} and it would do the same thing.

Reply:
Perhaps my confusion is coming from `12` meaning "fill the available space". From my experience it means "full-width column, breaking on a new line if necessary"

Reply:
I agree with @enstyled here, without looking at the prop table, I'd say that 12 is a "full-width column" example

Reply:
Updated the example to include a span value.

Reply:
Added short blurb about this.